src/commands/link.ts

```diff
@@ -3,30 +3,32 @@ import { flags as oclifFlags } from '@oclif/command'
 import { CustomCommand } from '../api/oclif/CustomCommand'
 import { appLink } from '../modules/apps/link'
 
+import { ColorifyConstants } from '../api/constants/Colors'
+
 export default class Link extends CustomCommand {
-  static description = 'Start a development session for this app'
+  static description = `Locally syncs the app in the current directory with the ${ColorifyConstants.ID('development workspace')} in use.`
 
-  static examples = ['vtex link -a youraccount -w yourworkspace']
+  static examples = [`${ColorifyConstants.COMMAND_OR_VTEX_REF('vtex link')} -a youraccount -w yourworkspace`]
 
   static flags = {
     ...CustomCommand.globalFlags,
     account: oclifFlags.string({
       char: 'a',
-      description: `Account to login before linking the app. This flag has to be paired with the '--workspace' flag.`,
+      description: `Starts a development session in the specified ${ColorifyConstants.ID('account')}. Must be paired with the ${ColorifyConstants.COMMAND_OR_VTEX_REF('--workspace')} flag.`,
       required: false,
       dependsOn: ['workspace'],
     }),
-    clean: oclifFlags.boolean({ char: 'c', description: 'Clean builder cache', default: false }),
+    clean: oclifFlags.boolean({ char: 'c', description: 'Cleans builder cache', default: false }),
     setup: oclifFlags.boolean({
       char: 's',
-      description: 'Setup typings before linking [see vtex setup --help]',
+      description: `Sets up typing definitions before linking the app [see ${ColorifyConstants.COMMAND_OR_VTEX_REF('vtex setup --help')}].`,
```

Comment:
Here we can say that the setup happens locally

Reply:
```suggestion
      description: `Locally sets up typing definitions before linking the app [see ${ColorifyConstants.COMMAND_OR_VTEX_REF('vtex setup --help')}].`,
```